tests/markdown/MarkdownfmtBearTest.py

```diff
@@ -0,0 +1,37 @@
+from bears.markdown.MarkdownfmtBear import MarkdownfmtBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+header_in_file1 = """
+MarkdownBear
+=====
+
+"""
+
+header_out_file1 = """
+MarkdownBear
+============
+
+"""
+
+spacing_in_file2 = """
+
+This is   a test    file.
+
+"""
+
+spacing_out_file2 = """
+
+This is a test file.
+
+"""
+
+MarkdownfmtBear = verify_local_bear(
+    MarkdownfmtBear,
+    valid_files=(header_in_file1,),
+    invalid_files=(header_out_file1,))
+
+MarkdownfmtBear = verify_local_bear(
+    MarkdownfmtBear,
+    valid_files=(spacing_in_file2,),
+    invalid_files=(spacing_out_file2,))
```

Comment:
You need to improve your tests :wink: . You can refer to any bear with code correcting capabilities. (e.g. `MarkdownBear`)